src/index.scss

```diff
@@ -1,29 +0,0 @@
-$fa-font-path: "~font-awesome/fonts";
-@import "~font-awesome/scss/font-awesome";
-
-@import "~@edx/brand/paragon/fonts";
-@import "~@edx/brand/paragon/variables";
-@import "~@edx/paragon/scss/core/core";
-@import "~@edx/brand/paragon/overrides";
-
-@import "~@edx/frontend-component-header/dist/index";
-@import "~@edx/frontend-component-footer/dist/footer";
-
-@import "./console/style";
-
-.word-break-all {
-  word-break: break-all !important;
-}
-
-#app {
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  & > main {
-    flex-grow: 1;
-  }
-}
-
-footer a img {
-  max-height: 50px;
-}
```

Comment:
I don't see a replacement for this.

Reply:
@xitij2000 As far as I could gather, I could not see the use of this inside of code.